### PR TITLE
New Cask messenger-native

### PR DESCRIPTION
**Pull request by hanxue**

### CODE REVIEW

File: Casks/messenger-native.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'messenger-native' do
+  version '1.0.0'
+  sha256 '72d730197b91963a5804adf09cbb2a4df3d9374f0ec922035e34587d0e2c073a'
+
+  url "https://github.com/imton/MessengerNative/releases/download/1.0.0/Mac64_MessengerNative1.0.0.zip"
```
Comment (alebcay) [inline Casks/messenger-native.rb]: Try replacing this with `url "https://github.com/imton/MessengerNative/releases/download/#{version}/Mac64_MessengerNative#{version}.zip"`. Considering it's the first release, the naming scheme could change but we will adapt this later if we need to.

File: Casks/messenger-native.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  homepage 'https://github.com/imton/MessengerNative'
+  license :mit
+
+  app "osx64/Messenger Native.app"
```
Comment (alebcay) [inline Casks/messenger-native.rb]: Use single quotes, not double. No string interpolation is occurring.

```ruby
app 'osx64/Messenger Native.app'
```

File: Casks/messenger-native.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/messenger-native.rb]: An [appcast can also be added](https://github.com/caskroom/homebrew-cask/pull/10056/files).

File: Casks/messenger-native.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'messenger-native' do
+  version '1.0.0'
+  sha256 '72d730197b91963a5804adf09cbb2a4df3d9374f0ec922035e34587d0e2c073a'
+
+  url "https://github.com/imton/MessengerNative/releases/download/1.0.0/Mac64_MessengerNative1.0.0.zip"
+  homepage 'https://github.com/imton/MessengerNative'
```
Comment (vitorgalvao) [inline Casks/messenger-native.rb]: Homepage is `https://www.messenger.com/`.

File: Casks/messenger-native.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/messenger-native.rb]: An [appcast can be added](https://github.com/caskroom/homebrew-cask/pull/10056/files).

Also needs a `# github.com is the official download host per the vendor homepage` comment above, after the `hompeage` change.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It will be merged in the pull request referenced above due to some simple recommended alterations. Your contribution is still included (and still credited to you), with the appropriate modifications, when applicable.

**Pull request closed without merging**